gnosis/cowswap
# Feature/1730 notif banner miss adjusts

# Summary

Fixes #1730 

Notification banner is expected to disappear after it was closed and a page was reloaded.

  # To Test

To reproduce:
1. Connect a wallet in the Mainnet with the app
2. Open profile page and copy a referral link
3. Paste the link into the browser
4. Close the info banner
5. Refresh the page

AR: banner appears again

ER: banner should not appear when it is closed for the current account
Need to show banner again when switch to another account
